grid/social-media/publish-post/maps/facebook.test.ts

```diff
@@ -0,0 +1,43 @@
+import { RecordingDefinitions } from '@superfaceai/testing';
+import { env } from 'process';
+
+import { publishPostTest } from './publish-post';
+
+const ACCESS_TOKEN_REDACTED_VALUE = 'PARAMS_pageAccessToken';
+
+const beforeRecordingSave = function (recordings: RecordingDefinitions) {
```

Comment:
Would be helpful to explain rationale for this recordings filter, e.g.:

```
// Remove page access tokens from internal requests and responses
```

Reply:
added